src/BSON/Binary.c

```diff
@@ -280,6 +280,9 @@ ZEND_BEGIN_ARG_INFO_EX(ai_Binary___set_state, 0, 0, 1)
 	ZEND_ARG_ARRAY_INFO(0, properties, 0)
 ZEND_END_ARG_INFO()
 
+ZEND_BEGIN_ARG_WITH_TENTATIVE_RETURN_TYPE_INFO_EX(ai_Binary_jsonSerialize, 0, 0, IS_ARRAY, 0)
+ZEND_END_ARG_INFO()
+
```

Comment:
Since these are reused, can we declare these `ai_*_jsonSerialize` variables somewhere else and reuse them? The signature of these methods is fixed by the interface, so technically we could have this definition in the same place. Happy to defer this to a follow-up PR if you'd like to get it fixed first.

Reply:
Deferring:

 * https://jira.mongodb.org/browse/PHPC-1938
 * https://jira.mongodb.org/browse/PHPC-1939